src/test/ui/symbol-names/impl1.rs

```diff
@@ -75,3 +75,7 @@ fn main() {
         }
     };
 }
+
+// FIXME(katie): The 32-bit symbol hash probably needs updating as well, but I'm slightly unsure
```

Comment:
you should be able to cross-compile I guess, but another option is just to let CI tell you the right answer